integration/poetry_run_test.go

```diff
@@ -94,6 +95,7 @@ func testPoetryRun(t *testing.T, context spec.G, it spec.S) {
 			Expect(logs).To(ContainLines(ContainSubstring("Poetry Run Buildpack")))
 			Expect(logs).To(ContainLines(ContainSubstring("Environment Variables Buildpack")))
 			Expect(logs).To(ContainLines(ContainSubstring("Image Labels Buildpack")))
+			Expect(logs).To(ContainLines(ContainSubstring("Watchexec Buildpack")))
 
 			Expect(image.Buildpacks[6].Key).To(Equal("paketo-buildpacks/environment-variables"))
```

Comment:
The indices here just need to be incremented, then it should be good to go.

Reply:
Ah, good catch. I saw that possibility but ignored it because I thought watchexec was last in the last (i.e. after the env vars). I'll make that change.